x-pack/legacy/plugins/uptime/public/components/functional/location_map/location_missing.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState } from 'react';
+import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiPopover, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { LocationLink } from '../monitor_list/monitor_list_drawer';
+
+export const LocationMissingWarning = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  const togglePopover = () => {
+    setIsPopoverOpen(!isPopoverOpen);
+  };
+
+  const button = (
+    <EuiButton iconType="alert" size="s" color="warning" onClick={togglePopover}>
+      Geo Information Missing
+    </EuiButton>
+  );
+
+  return (
+    <EuiFlexGroup gutterSize="none">
+      <EuiFlexItem grow={false} style={{ marginLeft: 'auto' }}>
+        <EuiPopover
+          id="popover"
+          button={button}
+          isOpen={isPopoverOpen}
+          closePopover={togglePopover}
+        >
+          <EuiText style={{ width: '300px' }}>
```

Comment:
I think we can improve the formatting/messaging here a bit. This isn't perfect, but something along these lines:

```typescript
          <EuiText style={{ width: '300px' }}>
            <FormattedMessage
              id="xpack.uptime.locationMap.locations.missing.message"
              defaultMessage="Important geo location configuration is missing. You can use the {test} field
            to create distinctive geographic regions for your uptime checks."
              values={{ test: <EuiCode>observer.geo.??</EuiCode> }}
            />
          </EuiText>
          <EuiSpacer size="xs" />
          <EuiText style={{ width: '300px' }}>
            <FormattedMessage
              id="xpack.uptime.locationMap.locations.missing.message1"
              defaultMessage="Get more information in
            our documentation."
            />
          </EuiText>
          <EuiSpacer size="xs" />
          <LocationLink />
```

Renders something like this:
![image](https://user-images.githubusercontent.com/18429259/72290052-659c1980-361a-11ea-9701-ff84eda4ed7a.png)

WDYT?